build.sbt

```diff
@@ -39,13 +39,7 @@ lazy val webapp = Project(
   settings = buildSettings
 ).enablePlugins(LibraryPlugin).dependsOn(core, testkit % "test->compile")
 
-lazy val pipeline = Project(
```

Comment:
I'd expect you already have the https://github.com/allenai/pipeline repo. If you have that, then I'd expect this PR to replace the deleted code with a new Maven-based jar. So everything keeps working...